website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4775,12 +4795,15 @@ _ = (p | 'Read per user' >> ReadPerUser()
 
 #### 11.3.3. Dynamic timer tags {#dynamic-timer-tags}
 
-Beam also supports dynamically setting a timer tag using `TimerMap`. This allows for setting multiple different timers
+Beam also supports dynamically setting a timer tag using `TimerMap` in Java SDK. This allows for setting multiple different timers
 in a `DoFn` and allowing for the timer tags to be dynamically chosen - e.g. based on data in the input elements. A
 timer with a specific tag can only be set to a single timestamp, so setting the timer again has the effect of
 overwriting the previous expiration time for the timer with that tag. Each `TimerMap` is identified with a timer family
 id, and timers in different timer families are independent.
 
+In Python SDK, a dynamic timer tag can be specified while calling timer.set() or timer.clear(), by default the timer
```

Comment:
`set()` and `clear()`

Unless you think it's too ambiguous without the class name. If that's the case, then add these in code font.

Reply:
Done.